app/src/main/java/at/tacticaldevc/oat/ui/TrustedNumbers/Contact.java

```diff
@@ -12,7 +12,7 @@
     private String name, number;
 
     public Contact(String name, String number) {
-        ensureStringIsValid(name, "name");
+        ensureStringIsValid(name, name);
```

Comment:
The second argument should be the name of the first argument for easier debugging.

Reply:
I see, i will change it back

Reply:
+1 just leave it with "name"

Reply:
Otherwise we might get a exception telling us that null should not be null xD

Reply:
I just pushed it.